strato_query/__tests__/test_api_query.py

```diff
@@ -809,7 +789,7 @@ def test_job_runner_with_portfolio_time_series(self):
         df = SDJobRunner().load_df_from_job_pipeline(
             # Same job
             portfolio_id='WMqA42Dw',
-            model_id='XYjooEDw',  # This is a time series query
+            model_id='wD1Ga87A',  # This is a time series query
```

Comment:
What's the update here for the model id?

Reply:
I think the model the old ID referenced was deleted, because the test kept failing. The new model is still a time series of employment, shared with SD, and it is now passing without issue.